.travis.yml

```diff
@@ -25,39 +26,54 @@ os:
 # - windows # Windows is not supported at all by Travis CI as of Feb. 2018
 
 # Linux setup
-dist: trusty
+dist: xenial
 sudo: required
 
 matrix:
-  # see "os: ..." above
   include:
+    # building the docs
+    - python: "3.7"
+      env: BUILD_ONLY_DOCS=TRUE
+    # testing socketcan on Trusty & Python 3.6, since it is not available on Xenial
+    - os: linux
```

Comment:
It might just be this one that needs sudo?

Reply:
Ah yes, that could work.